iostreams/iostreams.go

```diff
@@ -188,7 +188,7 @@ func (s *IOStreams) IsStdoutTTY() bool {
 	}
 
 	// support KRAFTKIT_FORCE_TTY
-	if s.term.IsTerminalOutput() {
+	if s.term != nil && s.term.IsTerminalOutput() {
```

Comment:
I didn't dig into why this happens when nesting ``processtrees``, but maybe this is not the way to handle this seeing as this check is not performed anywhere else.

I'm not sure whether this is fine or it means somewhere else needs fixing.